device-types/Ubiquiti/USW-Pro-Max-48.yaml

```diff
@@ -0,0 +1,131 @@
+---
+manufacturer: Ubiquiti
+model: UniFi Switch Pro Max 48
+slug: ubiquiti-unifi-switch-pro-max-48
+part_number: USW-Pro-Max-48
+comments: |
+  UniFi Switch 48 Pro Max (non-PoE) with total non-blocking throughput of 112 Gbps. Features (32) Gigabit Ethernet ports, (16) 1/2.5GbE ports, and (4) 10G SFP+ uplink ports.
+
+  Dimensions: 442 x 325 x 44 mm (17.4 x 12.8 x 1.7")
+u_height: 1
+weight: 4.9
+weight_unit: kg
+airflow: front-to-rear
+is_full_depth: false
+front_image: true
+rear_image: true
+interfaces:
+  # Gigabit Ethernet ports (1-32)
+  - name: Port 1
+    type: 1000base-t
+  - name: Port 2
+    type: 1000base-t
+  - name: Port 3
+    type: 1000base-t
+  - name: Port 4
+    type: 1000base-t
+  - name: Port 5
+    type: 1000base-t
+  - name: Port 6
+    type: 1000base-t
+  - name: Port 7
+    type: 1000base-t
+  - name: Port 8
+    type: 1000base-t
+  - name: Port 9
+    type: 1000base-t
+  - name: Port 10
+    type: 1000base-t
+  - name: Port 11
+    type: 1000base-t
+  - name: Port 12
+    type: 1000base-t
+  - name: Port 13
+    type: 1000base-t
+  - name: Port 14
+    type: 1000base-t
+  - name: Port 15
+    type: 1000base-t
+  - name: Port 16
+    type: 1000base-t
+  - name: Port 17
+    type: 1000base-t
+  - name: Port 18
+    type: 1000base-t
+  - name: Port 19
+    type: 1000base-t
+  - name: Port 20
+    type: 1000base-t
+  - name: Port 21
+    type: 1000base-t
+  - name: Port 22
+    type: 1000base-t
+  - name: Port 23
+    type: 1000base-t
+  - name: Port 24
+    type: 1000base-t
+  - name: Port 25
+    type: 1000base-t
+  - name: Port 26
+    type: 1000base-t
+  - name: Port 27
+    type: 1000base-t
+  - name: Port 28
+    type: 1000base-t
+  - name: Port 29
+    type: 1000base-t
+  - name: Port 30
+    type: 1000base-t
+  - name: Port 31
+    type: 1000base-t
+  - name: Port 32
+    type: 1000base-t
+  # 2.5GbE Ethernet ports (33-48)
+  - name: Port 33
+    type: 2.5gbase-t
+  - name: Port 34
+    type: 2.5gbase-t
+  - name: Port 35
+    type: 2.5gbase-t
+  - name: Port 36
+    type: 2.5gbase-t
+  - name: Port 37
+    type: 2.5gbase-t
+  - name: Port 38
+    type: 2.5gbase-t
+  - name: Port 39
+    type: 2.5gbase-t
+  - name: Port 40
+    type: 2.5gbase-t
+  - name: Port 41
+    type: 2.5gbase-t
+  - name: Port 42
+    type: 2.5gbase-t
+  - name: Port 43
+    type: 2.5gbase-t
+  - name: Port 44
+    type: 2.5gbase-t
+  - name: Port 45
+    type: 2.5gbase-t
+  - name: Port 46
+    type: 2.5gbase-t
+  - name: Port 47
+    type: 2.5gbase-t
+  - name: Port 48
+    type: 2.5gbase-t
+  # SFP+ uplink ports (49-52)
+  - name: SFP+ 49
+    type: 10gbase-x-sfpp
+  - name: SFP+ 50
+    type: 10gbase-x-sfpp
+  - name: SFP+ 51
+    type: 10gbase-x-sfpp
+  - name: SFP+ 52
+    type: 10gbase-x-sfpp
+power-ports:
+  - name: AC Input
+    type: iec-60320-c14
+    maximum_draw: 100
+  - name: USP-RPS Input
+    type: iec-60320-c20  # Updated to reflect standard support
```

Comment:
This is not a iec-60320-c20 port please use 'ubiquiti-smartpower'

Reply:
I updated this to the ubiquiti-smartpower.